_pages/openmic.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: "Open mic sessions"
+layout: single
+permalink: open-mic
+taxonomy: OpenMic
+
+lang: en
+lang-ref: open-mic
+#classes: wide
+author_profile: true
+
+header:
+  image: /assets/images/archive/openmic.jpg
+  caption: "Photo credit: [**Matthias Wagner**](https://unsplash.com/@matwag?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
+---
+
+Open mic session is an activity going on irregulary every two weeks. A member of a team or a guest presents an interesting topic to the rest of the team and anyone willig to listen.
```

Comment:
"going on irregularly every two weeks" - that sounds like an oxymoron. Maybe something like "going on irregularly with approximately two week period"? Or omit the two week altogether?